cloudbuild/e2e-test-scripts/table_read.sh

```diff
@@ -36,7 +36,9 @@ if [ "$MODE" == "bounded" ]
 then
   echo "Bounded Mode!"
   source cloudbuild/e2e-test-scripts/bounded_table_read.sh
-
+else
+  echo "Unbounded Mode!"
+  source cloudbuild/e2e-test-scripts/unbounded_table_read.sh
```

Comment:
We should explicitly check if mode is `unbounded`, just like the `bounded` case. If mode is neither bounded nor unbounded, throw error.